common/snowflake/Makefile

```diff
@@ -111,6 +114,15 @@ ifeq ($(SF_SHARE_ENABLED), 1)
 	$(SED) $(REPLACEMENTS) $(SHARE_REMOVE_FILE) | $(SNOWSQL) -q "$(xargs)" 
 endif
 
+serialize-module:
```

Comment:
this should be added to `.PHONY`I guess.

Reply:
You're right!